tests/rbd/rbd_failback.py

```diff
@@ -0,0 +1,85 @@
+import time
+
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verification of rbd mirroring failback scenarios on primary site,
+    This module configures the mirroring back to failback cluster.
+
+    Args:
+        kw: test data
+
+    Returns:
+        int: The return value - 0 for success, 1 otherwise
+
+    Test case covered -
+    CEPH-9476- Primary cluster permanent failure,
+    Recreate the primary cluster and Re-establish mirror with newly created cluster as Primary.
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+    (At least with 64 pgs)
+    2. We need atleast one client node with ceph-common package,
+    conf and keyring files
+
+    Test case flows:
+    1) Create a pool with same name as secondary pool has
```

Comment:
Are you passing poolname as a parameter? How would you know which pool secondary has?

Reply:
yes in the updated PR using common pool name for mirror test